packages/pds/src/db/records/vote.ts

```diff
@@ -89,21 +92,25 @@ const notifsForRecord = (
   if (!matchesSchema(obj)) {
     throw new Error(`Record does not match schema: ${type}`)
   }
+  if (obj.direction === 'down') {
+    // No notifications for downvotes
```

Comment:
I actually wonder if we _should_ return downvote notifs from the server & leave it up to the client if they want to display them 🤔 

Gives the option to filter them out, aggregate them into a summary, or just ruin the user's day by informing them of every downvote 😛

Reply:
@pfrazee any feels on this?  Do you want downvote notifications from the backend and allow clients to filter them as needed?  I think it could still mess with unread counts, and maybe create some new cases to handle around setting the latest unseen time.